commands/github-info.js

```diff
@@ -1,34 +1,47 @@
-const fs = require("fs");
+const db = require("../database");
+const { Octokit } = require("@octokit/core");
+const { restEndpointMethods } = require("@octokit/plugin-rest-endpoint-methods");
+const MyOctokit = Octokit.plugin(restEndpointMethods);
 
 module.exports = {
-    name: "github-info",
-    description: "Get GitHub Personal Access Token",
-    execute(command, message, args) {
-        let githubToken = null;
-
-        function writeToken(githubToken) {
-            try {
-                console.info("GitHub token is " + githubToken);
-                const data = fs.writeFileSync("./github-token.txt", githubToken);
-            } catch (err) {
-                console.error(err);
-                return message.reply(
-                    "GitHub auth was unsuccessful. Please try again."
-                );
-            }
-        }
-
-        // -github-info: Get contents of personal token
-        if (command === "github-info") {
-            if (!args.length) {
-                return message.reply("you didn't provide a GitHub Personal Token.");
-            } else {
-                githubToken = args[0];
-                writeToken(githubToken);
-                return message.reply(
-                    "GitHub auth was successful. Use -github-issue-number with the number of the issue you'd like to comment on."
-                );
-            }
-        }
-    },
-};
+  name: "github-info",
+  description: "Get GitHub Personal Access Token",
+  usage: "-github-info <your-github-personal-access-token>",
+  execute(command, message, args) {
+    if (!args.length) {
+      db.fetchGit(message.author.id)
+        .then((result) => {
+          return message.reply(`Your GitHub token is on file: ${result}`);
+        })
+        .catch((err) => {
+          console.error(err);
+          return message.reply(
+            "Your GitHub token isn't on file. Try again, specifying your GitHub token (once you do it, we'll store it securely)"
+          );
+        });
+    } else {
+      let githubToken = args[0];
```

Comment:
What happens if no argument is passed? Accessing `args[0]` would then throw an error